### DISCUSSION TITLE
"SSL Certificate Expires in xx days" Still shows after Updating Certs

### DISCUSSION THREAD

**Discussion by modem-19** [amidaware/tacticalrmm#1797] [General]
I just wanted to check and see what I might be doing wrong.  Earlier this month I got the "SSL certificate expires in xx days" error message.  This past week I have been working at getting this fixed but so far the warning still shows in my portal.  

I followed the instructions fully here at this tacticalRMM doc's page by running the certbot command so that all the various subdomains are updated with the latest certificates.  Keep in mind, my RMM (test platform) resides on a sub-domain of https://rmm.mydomain.com.  So when I created DNS records, I created "_acme-challenge.rmm.mydomain.com  3600  TXT xxxxxxxxxxxxxxxx"  Along with mesh and api.

After that was done I ran the update.sh --force command.  Afterwards, no change.

For the record, I'm on v0.17.5.

Suggestions? Thoughts?  Input?

**Comment by wh1te909** [comment]
you need to get a wildcard cert, not for individual subdomains
your certbot command should be for `*.mydomain.com` and your acme challenge should be `_acme-challenge.mydomain.com`

**Reply by modem-19** [reply]
@wh1te909 Thanks for the heads up on that.  I mistook it as having to do it for each of the subdomains, but using it as you mentioned solved the issue.  Thanks for the assistance!
